new/LIT7247QeneSemaetena.xml

```diff
@@ -0,0 +1,71 @@
+                <title xml:lang="en" corresp="#t1">Our martyr Claudius, poor in time and days…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Neither type nor author of the qǝne are indicated in the manuscript. According to the classification by 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl> a qǝne possibly of the 
+                    wāzema-type.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Wazema"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-01-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/>, fol. 56rb. Division into verses is indicated with ። . 
+                    The end of the poem is indicated with ። ። .</note>
+                <ab>
+                    <l n="1">ሰማዕትነ፡ ገላውዴዎስ፡ ነዳየ፡ ዘመን፡ ወመዋዕል፡ እንዘ፡ ይረውፅ፡ አልፀቀ፡ እስከ፡ አንቀጸ፡ ቄሳር፡ ገድል።</l> 
+                    <l n="2">ወተርኅወ፡ ሎቱ፡ ሙባኡ፡ መስቀል።</l>
+                    <l n="3">ኀበ፡ ፍርፋረ፡ ሞት፡ ተርኅወ፡ ወስቴ፡ መከራ፡ ጥሉል።</l>
+                    <l n="4">አፃዌ፡ ኆኅት፡ አማኑኤል፡ አኮኑ፡ ቀዊሞ፡ በዴዴ፡ አስካል።</l>
+                    <l n="5">ዘርኅብከ፡ ዘርኅብከ፡ ይብል። ።</l>
+                </ab>
+            </div>
+        </body>
```

Comment:
it is OK just a comment - when we know who poems/texts are dedicated to it is nice to have the is_about relation, so that from the persons one could easily see all texts dedicated to them (PRS3126Claudius)